examples/femnist.py

```diff
@@ -132,6 +132,9 @@ def main():
 
     args.num_clients = len(train_datasets)
     model = get_model(args)
+    cfg.num_gpu = args.num_gpu
+    if cfg.num_gpu > 1:
+        model = torch.nn.DataParallel(model) 

```

Comment:
I wonder why we use DataParallel here? From its [document](https://pytorch.org/docs/stable/generated/torch.nn.DataParallel.html), it says: It is recommended to use [DistributedDataParallel](https://pytorch.org/docs/stable/generated/torch.nn.parallel.DistributedDataParallel.html#torch.nn.parallel.DistributedDataParallel), instead of this class, to do multi-GPU training, even if there is only a single node. See: [Use nn.parallel.DistributedDataParallel instead of multiprocessing or nn.DataParallel](https://pytorch.org/docs/stable/notes/cuda.html#cuda-nn-ddp-instead) and [Distributed Data Parallel](https://pytorch.org/docs/stable/notes/ddp.html#ddp).

Reply:
@Zilinghan The DataParallel is the legacy code for testing the Torch model-level parallelism. As APPFL assigns the model on the single-process, and single thread in a multi-process environment with MPI, APPFL does not need to implement the DataParallel and DistributedDataParallel.

Reply:
So will this be removed as mentioned below?